homeassistant/components/samsungtv/config_flow.py

```diff
@@ -227,7 +221,7 @@ def _async_update_existing_host_entry(self) -> config_entries.ConfigEntry | None
         """
         for entry in self._async_current_entries(include_ignore=False):
             if entry.data[CONF_HOST] != self._host:
-                continue
+                continue  # pragma: no cover
```

Comment:
Added `# pragma: no cover` to ensure 100% coverage.
According to discord conversation, this was discussed between @chemelli74 and @bdraco.

Reply:
Ideally we cover this line.  We can merge it without the `no cover` and fix it later since it is an existing issue.